src/components/ErrorLogsView/utils/generateAffectedRecordInfo.js

```diff
@@ -75,6 +77,20 @@ const generateAffectedRecordLevel = ({
     ));
   }
 
+  if (record.inventoryRecordLink) {
```

Comment:
Let's add a test to check that the link contains the correct `href` and text.

Reply:
I'll do this in a bit, I just created PR to check whether CI is passing.

Reply:
Yeah, I added this comment before you even assigned me:)

Reply:
You're very impatient 😉